roles/keystone/meta/main.yml

```diff
@@ -5,3 +5,9 @@ dependencies:
     service: keystone
     logdata: "{{ keystone.logs }}"
   - role: openstack-source
+    project_name: keystone
+    project_rev: "{{ keystone.source.rev }}"
+    # is this even required anymore with Juno?
+    python_requirements:
+      - { name: babel, version: "{{ keystone.babel_version }}" }
```

Comment:
Great question here about juno. In fact, Babel is now listed in the requirements.txt file, so we can remove all the babel shenanigans. Good catch.